packages/console/src/types/spaces.ts

```diff
@@ -0,0 +1,35 @@
+export interface SpaceWithAccessType {
+  accessType?: 'public' | 'private'
+  // ... other space properties from @storacha/ui-react Space interface
+}
+
+export interface PrivateSpacesFeatureFlags {
+  canAccessPrivateSpaces: boolean
+  shouldShowUpgradePrompt: boolean
+  shouldShowPrivateSpacesTab: boolean
```

Comment:
my review of the code is this is never referenced, and more over, it is entirely duplicative of canSeePrivateSpacesFeature on the feature flags struct